lib/openshift/kiali.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class Kiali < ProjectResource
```

Comment:
Here it indicates it is a project resource, but the first line is requiring cluster resource.  Need to update them in sync.

Reply:
OK, I copied them from etcd_cluster.rb. Will update.